dts/bindings/clock/refsoc,clock-control.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
missing copyrights in some new yaml files

Reply:
Fixed.Thanks for the review!